server/controllers/Profiles.js

```diff
@@ -19,7 +19,10 @@ class Profiles {
   static async update(req, res) {
```

Comment:
Please undo all the changes in this method. Sad that I have to keep talking about this

Reply:
The changes occurred due to merge conflicts